lib/utilization/common.js

```diff
@@ -38,8 +38,12 @@ exports.getKeys = function getKeys(data, keys, allValuesMustBeValid = true) {
     const key = keys[i]
     if (!properties.hasOwn(data, key) || !data[key]) {
       logger.debug('Key %s missing from metadata', key)
-      return null
+      if (allValuesMustBeValid) {
+        return null
+      }
+      continue
     }
+
     let value = data[key]
```

Comment:
Would it be possible to add tests for these uncovered lines? (I'd think asserting the intended behavior for `allValuesMustBeValid = true` and for `false`.)